packages/plugins/other/visitor-plugin-common/tests/parse-mapper.spec.ts

```diff
@@ -68,4 +68,75 @@ describe('parseMapper', () => {
       source: 'file',
     });
   });
+
+  describe('suffix', () => {
+    it('Should return the correct values for a simple named mapper', () => {
+      const result = parseMapper('MyType', null, 'Model');
+
+      expect(result).toEqual({
+        isExternal: false,
+        type: 'MyType',
+      });
+    });
+
+    it('Should return the correct values for a external named mapper', () => {
+      const result = parseMapper('file#Type', null, 'Model');
+
+      expect(result).toEqual({
+        default: false,
+        isExternal: true,
+        import: 'Type as TypeModel',
+        type: 'TypeModel',
+        source: 'file',
+      });
+    });
+
+    it('Should return the correct values for a external default mapper', () => {
+      const result = parseMapper('file#default', 'MyGqlType', 'Model');
+
+      expect(result).toEqual({
+        default: true,
+        isExternal: true,
+        import: 'MyGqlType',
+        type: 'MyGqlType',
+        source: 'file',
+      });
+    });
```

Comment:
I'm not completely sure if this is the desired outcome; the `MyGqlType` is a type that's derived from the schema, right?